iroh-net/src/magic_endpoint.rs

```diff
@@ -971,4 +990,70 @@ mod tests {
         p1_connect.await.unwrap();
         p2_connect.await.unwrap();
     }
+
+    #[tokio::test]
+    async fn magic_endpoint_wait_for_direct_conn() {
+        let _logging_guard = iroh_test::logging::setup();
+        let (relay_map, relay_url, _relay_guard) = run_relay_server().await.unwrap();
+        let mut rng = rand_chacha::ChaCha8Rng::seed_from_u64(42);
```

Comment:
🤣 TBH, this was taken from another test & I didn't change it. I'm assuming it was originally 42 because 42 is the answer to life the universe and everything